### PR TITLE
nuccChunkModelHit support

### PR DESCRIPTION
**Pull request by maxcabd**

### CODE REVIEW

File: CyberConnect2/nucc.bt [hunk 1]
```diff
@@ -133,48 +134,52 @@ typedef struct
...
     {
         struct
         {
-            f32 Values[6];
+            f32 OffsetZ;
+            f32 OffsetY;
+            f32 OffsetX;
+            f32 ScaleZ;
+            f32 ScaleY;
+            f32 ScaleX;
```
Comment (mosamadeeb) [inline CyberConnect2/nucc.bt:166]: Are these coordinates actually reversed, or is this based on Blender's coordinate system?

The template should be in the game's coordinate system, so these should be XYZ unless they're actually ZYX in the game as well.

File: CyberConnect2/nucc.bt [hunk 2]
```diff
@@ -321,3 +326,52 @@ typedef struct
...
+                u16 FootEffect2;
+
+                if (ReadInt64() == 0) 
+                {
+                    FSkip(8);
+                }
+                else 
+                {
+                    FSkip(0);
+                }
```
Comment (mosamadeeb) [inline CyberConnect2/nucc.bt]: Is there a reason to not remove this?

**Review by mosamadeeb — COMMENTED** [review]
Thanks for the contribution. Please check the comments I left first.

File: CyberConnect2/nucc.bt (hunk 1 above)
Comment (maxcabd) [reply CyberConnect2/nucc.bt:166]: Not sure about this one. These were the changes Hydra made so I assume they were correct. Before using Blender he would use 010 editor to edit the values and it seems he figured they must be flipped.

File: CyberConnect2/nucc.bt (hunk 2 above)
Comment (maxcabd) [reply CyberConnect2/nucc.bt]: I'll remove it and update my changes.

**Comment by mosamadeeb** [comment]
Looks good, thanks!

**Pull request merged**